### DISCUSSION TITLE
Test Medium parte 1

### DISCUSSION THREAD

**Discussion by Michael-Rodriguez-Tapia** [IIC2133-PUC/2024-2#128] [Tarea 1] [answered]
Holaaa, Segun lo que analice el output esperado de este test está erróneo, ya que indica un pingüino como mayor, el cual no lo es:

Output esperado (Ln 43-54):
Penguin 10 happiness updated to 89
Festival Stats
	Place 0 has 3 penguin[s]
		46 1 0
	Place 1 has 7 penguin[s]
		82 5 15
	Place 2 has 5 penguin[s]
		63 8 24
	Place 3 has 4 penguin[s]
		92 9 23
	Place 4 has 6 penguin[s]
		55 12 17

INPUT:
ENTER 3 23 84
HAPPINESS_BUFF 3 9 30
HAPPINESS_BUFF 0 1 51
HAPPINESS_BUFF 3 10 24
ENTER 2 24 15
ENTER 1 25 60
FESTIVAL

Mi output:
Penguin 10 happiness updated to 89
Festival Stats
	Place 0 has 3 penguin[s]
		46 1 0
	Place 1 has 7 penguin[s]
		82 5 15
	Place 2 has 5 penguin[s]
		63 8 24
	Place 3 has 4 penguin[s]
		92 9 10
	Place 4 has 6 penguin[s]
		55 12 17

Como se puede ver cuando se ejecuta el último festival de mi extracto, el output esperado indica que el pingüino con mayor felicidad debería ser el 23 que llego recientemente; sin embargo, se mantiene el 10 al cual se le aumenta su felicidad recientemente en los eventos.

Escribo más que nada para ver si me pueden confirmar esta situación,

**Comment by Segismundios** [comment, marked as answer]
Holaa @Michael-Rodriguez-Tapia. La primera id indicada es el pingüino con mayor felicidad, la segunda id es el con menor felicidad. En el sector que indicas, el pingüino con mayor felicidad es el de id 9, que en ese momento tiene un valor de 112. Por otro lado, el pingüino con menor felicidad es el de id 23, con 84 de felicidad. Al parecer tu output no está reflejando el buff de felicidad al pingüino 10, e indica que es el que tiene menor felicidad, pese a que ahora es de 89.
